html/poems.html

```diff
@@ -62,7 +62,13 @@
 
     <div class="container">
 

```

Comment:
You need to a div class="row" inside the container div. Otherwise the placing of you content will be a bit off. 
I am not sure if you have noticed but the styling of the page  does not work some div are covering other ones making impossible to see if what you have done actually works.